lib/transformer/runtime.rb

```diff
@@ -0,0 +1,15 @@
+module Transformer
+  class Runtime < Atlas::Runtime
+    def initialize(dataset, cast)
+      @dataset = dataset
+      @cast    = cast
+      @graph   = Atlas::GraphBuilder.build
+    end
+
+    private
+
+    def ETLOCAL(key)
```

Comment:
Are you open to calling this something else? The name doesn't make it obvious what is returned. `GET(key)` might be clearer and is nice and short (although: "get what?"). `USER_VALUE(key)` or `CHOICE(key)` might also be options. Perhaps you have a better idea?